doc/source/user_guide/label_guide.rst

```diff
@@ -42,7 +42,7 @@ Example: Using the labeller argument
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can use the ``labeller`` argument to customize labels.
-Unlike the default labels that show ``theta``, not :math:`\theta` (generated from ``$\theta$`` using :math:`\LaTeX`), the ``labeller`` argument presents the labels with proper math notation.
+Unlike the default labels that show ``theta``, not :math:`\theta` (generated from ``\theta`` using :math:`\LaTeX`), the ``labeller`` argument presents the labels with proper math notation.
```

Comment:
I would keep the dolar because it's what we use in python to get matplotlib to use latex rendering, see MapLabeller object instatiation